searchresults/index.html

```diff
@@ -44,40 +41,37 @@ <h1>Search Results</h1>
           <p id="confirmation"> ..................</p>
         </li>
       </ol>
-    </form> -->
-
-<!-- todo: only show return flight when round trip was selected -->
-
-  <form id="flightselection" action="../seats/index.html" method="get">
-    <h2>Departing Flight</h2>
+    </form>
+    <form id="flightselection" action="../seats/index.html" method="get">
+      <h2>Departing Flight</h2>
   <!-- select departing flight -->
-    <ol>
-      <li>
-        <label for="flightone">Flight One</label>
-        <input type="radio" id="flightone" name="departflight" />
+  <ol id='Departflights'>
         <ul>
           <li>
-            Depature Time
+            <label>Flight One:<input type="radio" name="departflight" id="flightone"/></label>
           </li>
           <li>
-            Arrival Time
+            Depature Time 7:30 am
           </li>
           <li>
-            Flight Duration
+            Arrival Time 9:45 am
           </li>
           <li>
-            Stops
+            Flight Duration 2 hours and 15 min
           </li>
           <li>
-            Price
+            Stops 0
+          </li>
+          <li>
+            Price 540
           </li>
         </ul>
-      </li>
 
-      <li>
-        <label for="flighttwo">Flight Two</label>
-        <input type="radio" id="flighttwo" name="departflight" />
+  <!-- put the p inside of the value attribute instead??? -->
         <ul>
+          <li>
+          <label>Flight Two:<input type="radio" name="departflight" id="flighttwo"/></label>
```

Comment:
i'm not sure why exactly you changed this, but the way i had it before was correct. you put the input button inside of its own label and i think you did it because you're trying to get it all on the same line but i'm pretty sure that's not semantically correct so please put it back the way i had it before because there are other ways to make the things go on the same line without using potentially-wrong html